app/src/main/java/org/simple/clinic/recentpatient/AllRecentPatientsEffectHandler.kt

```diff
@@ -40,10 +42,12 @@ class AllRecentPatientsEffectHandler @AssistedInject constructor(
       effects
           .observeOn(schedulersProvider.io())
           .switchMap {
-            patientRepository
-                .recentPatients_old(currentFacility.get().uuid)
-                .map(::RecentPatientsLoaded)
+            pagerFactory.createPager(
+                sourceFactory = { patientRepository.recentPatients(facilityUuid = currentFacility.get().uuid) },
+                pageSize = 20
```

Comment:
Hmm, are we planning to hardcode these page sizes everywhere? AFAIK, we had extracted those out to be injected via Dagger.

Reply:
Another issue is that we are duplicating these magic numbers across both the tests and the production code. Maybe once we are done migrating fully to paging 3, we can refactor this out so it's defined in a single place.

Reply:
Usually, I am using firebase remote config for the page sizes. But the all recent patient screen seemed like it doesn't require that, so I hard-coded them. But we can extract out the page size to a config as we do with overdue and instant search. wdyt?

Reply:
>Another issue is that we are duplicating these magic numbers across both the tests and the production code. Maybe once we are done migrating fully to paging 3, we can refactor this out so it's defined in a single place.

Usually, for effect handler tests I reference the page size in the config we define. But for instrumentation tests that might be hard, since the page size is defined per feature. So, for example all recent patients can have a different page size compared to recent patients in home, so that page size varies in effect handler. But when testing repo the query is still same right? So we can probably extract out the page size constants in tests. wdyt?

Reply:
Another option we can use is, defining all the page sizes in `PagingModule` like so.
```kt
@Provides
@PagingSize(AllRecentPatients)
fun providesAllRecentPatientsPageSize(): Int {
  return 25
}
```

Reply:
Implemented the above approach, this seems like a good plan for keeping all the paging sizes in a single file.

Reply:
Yeah, this could work. Not really necessary to make this change in this PR, for what its worth. We could do this after migrating everything.